registry/src/skills.json

```
@@ -460,5 +460,51 @@
             "Synthesizing results",
             "Preparing insights"
         ]
+    }, 
+    {
+      "pluginId": "79449cd9-0590-4ccc-9d9f-12f99438c17b",
+      "name": "@moxie-protocol/plugin-degenfans-alfafrens",
+      "displayName": "DegenFans AlfaFrens",
+      "version": "0.0.1",
+      "author": "DegenFans",
+      "description": "Enables your agent to recommend staking options for AlfaFrens channels",
+      "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens",
+      "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens/images/logo.png",
+      "settings": {},
+      "capabilities": ["show the best staking options for a given staking amount"],
+      "starterQuestions": [
+        {
+          "label": "What are some staking recommendations you have for Alfafrens?",
+          "value": "What are some staking recommendations you have for Alfafrens?"
+        },
```

Adding this to starter questions

```suggestion
        {
          "label": "What is the Degenfans Alfafrens Skills?",
          "value": "What is the Degenfans Alfafrens Skills?"
        },
        {
          "label": "What are some staking recommendations you have for Alfafrens?",
          "value": "What are some staking recommendations you have for Alfafrens?"
        },
```